spec/pages/insights_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+RSpec.describe 'Fb::Page#insights' do
+  context 'given an invalid user access token' do
+    user = Fb::User.new 'invalid_token'
+    page = Fb::Page.new({"name"=>"test", "id"=>"1234", "user"=>user})
+
+    it 'raises Fb::Error' do
+      expect{page.insights}.to raise_error Fb::Error
+    end
+  end
+
+  context 'given a valid user access token and an existing Facebook page' do
+    user = Fb::User.new ENV['FB_TEST_ACCESS_TOKEN']
+    page = Fb::Page.new({"name" => "Games",
+      "id" => "872965469547237",
+      "user" => user})
+
+    it 'returns an array of metrics' do
+      expect(page.insights).to be_a(Array)
```

Comment:
@S1v4 Something else that could be interested to test is whether the order in which insights are returned by Facebook is the same in which metrics are provided. In other words if we ask for

```
page_views_total,page_fan_adds_unique,page_engaged_users,page_video_views
```

does that mean that the insights are returned exactly in that order?

If that's the case, then we should add a test, so we can rely on that order.
Otherwise, we would have to find a way to associate the resulting values with the metrics we asked for.